.github/workflows/javadocs.yml

```diff
@@ -53,9 +59,7 @@ jobs:
             cp -R fhir-parent/target/site/apidocs ../apidocs/
     - name: Commit and Add GH-Pages
       env:
-            GITHUB_TOKEN: ${{ secrets.DOCS_SITE_TOKEN }}
-            GITHUB_REPOSITORY: ${{ secrets.GITHUB_REPOSITORY }}
-            GITHUB_ACTOR: ${{ secrets.GITHUB_ACTOR }}
```

Comment:
according to their docs, github actions already set these environment variables (no need to access like a secret).  we should be able to confirm that as soon as this one gets merged :-)